docs/collaborate/community/index.md

```diff
@@ -5,36 +5,16 @@ We are always available in our slack channel and we really want that to be a blo
 Besides our slack channel we also have two formats for collaborating with the design systems team on a more structural level.
 
 
-## Warp Open Forum
-
-### Purpose
-
-For designers and engineers that are working with the design system.
-
-- Feedback and discussions in a larger setting
-- Warp team talks about new features
-- Low key and relaxed setting
-
-Warp representatives from design, web and apps will be there.
-This is the place to be if you want to keep up with Warp.
-
-
-### Target group
-
-Designers and engineers in NMP.
-
-The agenda can be found in the calendar invite, check [#nmp-warp-design-system](https://sch-chat.slack.com/archives/C04P0GYTHPV) for updates.
-
-*If you have questions about component usage please bring them to the Office hours instead.*
+## Warp Office Hours
 
+![Warp office hours every monday 14-15:30, 15-16:30 in Finland](/open-office.png)
 
 ### When?
 
-Every third Thursday of the month. Please sign up through the link in the calendar invite if you have a topic you want to discuss.
+Every Monday between 2 pm - 3:30pm (3-4:30pm in 🇫🇮Finland) we are hosting Warp office hours.
```

Comment:
Maybe we could agree on the format so it's the same everywhere 😄 2 pm, 2pm, 2 p.m., 2p.m., 2PM... Or maybe 14.00 like in the image?

Reply:
Decided to go with 14:00... 🙂